### DISCUSSION TITLE
Controlled Select with a nullable value

### DISCUSSION THREAD

**Discussion by fuma-nama** [radix-ui/primitives#2162] [Help] [answered]
I want to keep the select component to be controlled even with a null value.

The Select component doesn't allow a null value, only `string` or `undefined` (uncontrolled) are accepted.

For example, I expect that if the value is `null` (not `undefined`), it should display the placeholder.
However, it causes unexpected behavior and the placeholder doesn't even appears.

How can I achieve that, or are there some alternatives?

**Comment by benoitgrelard** [comment, marked as answer]
Related to #1569
